content/id/docs/concepts/storage/volumes.md

```diff
@@ -0,0 +1,1151 @@
+---
+title: Volumes
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Berkas-berkas yang disimpan di _disk_ di dalam Container bersifat tidak permanen (akan terhapus seiring dengan dihapusnya Container/Pod), yang menimbulkan beberapa masalah untuk aplikasi biasa saat berjalan di dalam Container. Pertama, saat sebuah Container mengalami kegagalan, Kubelet akan memulai kembali Container tersebut, tetapi semua berkas di dalamnya akan hilang - Container berjalan dalam kondisi yang bersih. Kedua, saat menjalankan banyak Container bersamaan di dalam sebuah `Pod`, biasanya diperlukan untuk saling berbagi berkas-berkas di antara Container-container tersebut. Kedua masalah tersebut dipecahkan oleh abstraksi `Volume` pada Kubernetes.
```

Comment:
```suggestion
Berkas-berkas yang disimpan di _disk_ di dalam Kontainer bersifat tidak permanen (akan terhapus seiring dengan dihapusnya Kontainer/Pod), yang menimbulkan beberapa masalah untuk aplikasi biasa saat berjalan di dalam Kontainer. Pertama, saat sebuah Kontainer mengalami kegagalan, Kubelet akan memulai kembali Kontainer tersebut, tetapi semua berkas di dalamnya akan hilang - Kontainer berjalan dalam kondisi yang bersih. Kedua, saat menjalankan banyak Kontainer bersamaan di dalam sebuah `Pod`, biasanya diperlukan untuk saling berbagi berkas-berkas di antara Kontainer-kontainer tersebut. Kedua masalah tersebut dipecahkan oleh abstraksi `Volume` pada Kubernetes.
```

Reply:
@irvifa For all Container to Kontainer translation. This is actually an old discussion with @girikuncoro 

When we refer to Containers in Kubernetes, shouldn't we actually use Container instead, as it is an [API object](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#container-v1-core)?

Reply:
this is true, we decided to use Container since it's kubernetes API object https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#container-v1-core

but the early translated pages still didn't follow this convention, which we have to fix sometime in the future